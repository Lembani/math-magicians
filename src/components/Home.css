.home-welcome {
  margin: 30px 0;
  font-size: 1.7rem;
  padding-left: 20px;
}

.para-one,
.para-two {
  padding: 0 20px;
  margin: 30px 0;
  line-height: 25px;
}
