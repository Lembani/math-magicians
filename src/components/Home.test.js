import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from './Home';

test('heading', () => {
    render(<Home />);
    const heading = screen.getByRole('heading');
    expect(heading).toBeInTheDocument();
});

test('First paragraph', () => {
    render(<Home />)
    const paragraphs = document.querySelector('.para-one');
    expect(paragraphs).toBeInTheDocument();
});

test('Second paragraph', () => {
    render(<Home />)
    const paragraphs = document.querySelector('.para-two');
    expect(paragraphs).toBeInTheDocument();
});

it('It renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON()
    expect(tree).toMatchSnapshot();
})
