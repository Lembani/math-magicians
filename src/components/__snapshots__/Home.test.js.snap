// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly 1`] = `
Array [
  <h2
    className="home-welcome"
  >
    Welcome to our page!
  </h2>,
  <p
    className="para-one"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ullam magnam, magni autem cupiditate consequatur tempore temporibus quia ab perferendis laudantium eos voluptates sint, saepe ipsum reprehenderit impedit consectetur at veniam architecto, culpa alias voluptatum itaque? Sint aperiam, beatae explicabo quis totam amet. Nihil praesentium, velit exercitationem animi, ipsum quibusdam eveniet distinctio nobis repudiandae laudantium sunt dolorum aliquam? Eius, aliquam! Architecto maiores explicabo facilis? Ullam repellat cumque delectus incidunt? Nam quam tempora doloribus sit, velit corrupti natus quis mollitia exercitationem dicta amet nihil molestias consequuntur vel consectetur vero eaque quas.
  </p>,
  <p
    className="para-two"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ullam magnam, magni autem cupiditate consequatur tempore temporibus quia ab perferendis laudantium eos voluptates sint, saepe ipsum reprehenderit impedit consectetur at veniam architecto, culpa alias voluptatum itaque? Sint aperiam, beatae explicabo quis totam amet. Nihil praesentium, velit exercitationem animi, ipsum quibusdam eveniet distinctio nobis repudiandae laudantium sunt dolorum aliquam? Eius, aliquam! Architecto maiores explicabo facilis? Ullam repellat cumque delectus incidunt? Nam quam tempora doloribus sit, velit corrupti natus quis mollitia exercitationem dicta amet nihil molestias consequuntur vel consectetur vero eaque quas.
  </p>,
]
`;
